Celeste.Mod.mm/Mod/Everest/Everest.Loader.cs

```diff
@@ -114,6 +115,16 @@ public static class Loader {
                 Tuple.Create("Nameguy's D-Sides", _VersionMax, "Monika's D-Sides", _VersionMax),
             };
 
+            /// <summary>
+            /// The path to the Everest /Mods/.everestignore file.
+            /// </summary>
+            public static string PathGlobalEverestIgnore { get; internal set; }
```

Comment:
I agree with Kalobi, I don't see a good use case for a global `.everestignore` and think it could cause more harm than good. Plus running recursive rule checks + regex on tens of thousands of assets at launch sounds like it could be a performace hit vs. the occasional mod-local one (hard to say w/o testing though)

Reply:
The point of the global `.everestignore` is to allow users to ignore stuff from mod creators who aren't providing their own for things they really should be. For example, if users want to hide demo/test maps. But yes, I agree it's too easy to cause issues that are difficult to track down, so I'll remove it.